fairer-iOS/fairer-iOS/Screens/Home/Home/ViewController/HomeViewController.swift

```diff
@@ -115,19 +89,28 @@ final class HomeViewController: BaseViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        
         self.setupDelegate()
         self.setWeekCalendarSwipeGesture()
         self.setButtonEvent()
         self.setDatePicker()
+        self.setNotification()
     }
     
     override func viewWillAppear(_ animated: Bool) {
-        self.addDummyData()
         self.getDivideIndex()
-        self.getWeekHouseWorks(
-            startDate: homeWeekCalendarCollectionView.fullDateList.first ?? String(),
-            endDate: homeWeekCalendarCollectionView.fullDateList.last ?? String()
-        )
+        
+        if homeWeekCalendarCollectionView.datePickedByOthers == "" {
+            self.getHouseWorksByDate(
+                startDate: homeWeekCalendarCollectionView.todayDateInString,
+                endDate: homeWeekCalendarCollectionView.todayDateInString
+            )
+        }else {
```

Comment:
p3;
```suggestion
        } else {
```
조그마한 띄어쓰기... 부탁드립니당 ㅎㅎ